packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -39,17 +39,16 @@ export type GPCPCDInitArgs = {
 };
 
 /**
- * Record argument type encapsulating a record of POD PCD arguments.
+ * Record argument type encapsulating a record of POD PCD arguments. The POD
+ * names used here must correspond to those used in the proof configuration.
  */
 export type PODPCDRecordArg = RecordArgument<
   PODName,
   PCDArgument<PODPCD, { notFoundMessage?: string; proofConfig?: string }>
```

Comment:
I didn't notice this before.  It seems wrong that proofConfig is here rather than on the RecordArgument, which is where you're specifying it in the example.  This controls the validatorParams?  Does it apply to both?  I didn't ever focus on the validatorParams being a specified type.  Maybe make this a named type so you can use it in both places, and also in the args of the validator function?